src/Miunie.Avalonia/ViewModels/MainWindowViewModel.cs

```diff
@@ -60,6 +61,12 @@ public string DiscordToken
             set => _ = this.RaiseAndSetIfChanged(ref _discordToken, value);
         }
 
+        public bool CommandsEnabled
+        {
+            get => _commandsEnabled;
+            set => _ = this.RaiseAndSetIfChanged(ref _commandsEnabled, value);
+        }
```

Comment:
I misunderstood the problem then. I have to change the botconfiguration a soon as the checkbox is checked/unchecked. Don't know how to combine that with the RaiseAndSet...
Is the following solution acceptable?

```
public bool CommandsEnabled
 {
            get => _miunie.BotConfiguration.CommandsEnabled;
            set => _miunie.BotConfiguration.CommandsEnabled = value;
 }
```

Reply:
It could look something like the following:
```
        public bool CommandsEnabled
        {
            get => _commandsEnabled;
            set
            {
                _miunie.BotConfiguration.CommandsEnabled = value;
                _ = this.RaiseAndSetIfChanged(ref _commandsEnabled, value);
            }
        }
```

Reply:
Yeah, but the _commandsEnabled var is pretty useless now.

Reply:
I pushed my changes without the _commandsEnabled var. It worked for me but if it is not the correct solution stylewise or otherwise I'll take another look at it tomorrow. Thanks for your feedback.